about.html

```diff
@@ -0,0 +1,196 @@
+        <ul class="connect">
+            <li class="s-icon1"><a href="#!"><i class="fa-brands fa-facebook"></i></a></li>
+            <li class="s-icon2"><a href="#!"><i class="fa-brands fa-twitter"></i></a></li>
+            <li class="lang"><a href="#!">Eng</a></li>
+            <li class="user-page"><a href="#!">My Page</a></li>
+            <li class="log-in-out"><a href="#!">Logout</a></li>
+       </ul>
+   </div>
+   <!--mobile nav-->
+   <nav class="primary-nav">
+    <!-- logo-->
+      <a href="./index.html" class="logo-container blur"> 
+        <!--logo image-->
+        <img id="logo" src="./images/cc-logo.png" alt="The CC logo">
+      </a> <!--logo ends-->
+      <!--mobile nav - hamburger-->
+      <a href="#" id="main-mobile-menu" class="hamburger-wrapper blur">
+        <img src="./images/mobile-menu.png" alt="Mobile menu icon">
+      </a>
+      <!--mobile menu items -->
+      <input type="checkbox" class="mobile-menu-nav-toggle" id="mobile-menu-nav-toggle">
+      <nav id="mobile-menu-nav" class="mobile-menu-nav">
+          <ul class="mobile-menu">
+              <li><a id="about" class="close-menu" href="./about.html">About</a></li>
+              <li><a id="program"  class="close-menu" href="./index.html">Program</a></li>
+              <li><a id="join"  class="close-menu" href="./index.html">Join</a></li>
+              <li><a id="news" class="close-menu" href="./index.html">News</a></li>
+              <li><a id="mcampaign" class="close-menu" href="./index.html">CC Campaign</a></li>
+            <label for="mobile-menu-nav-toggle" class="mobile-menu-nav-toggle-label">
+              <span>x</span>
+            </label>
+          </ul>
+      </nav>
+        <!--desktop menu-->
+      <ul class="desk-menu">
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">News</a></li>
+          <li id="dcampaign" class="campaign-link"><a href="#">CC Campaign</a></li>
+      </ul>
+    </nav> <!--header ends-->
+      <!--headline begins-->
+      <div class="abt-headline blur">
+        <p class="headline-greeting">"Hello! Changing the world"</p>
+        <hr class="headline-hr" />
+        <h1 class="headline-title abt-h1-title">
+          Creative Commons <br />
+          Global Summit 2013
+        </h1>
+        <p class="headline-intro abt-headline-intro">
+          The Global Summit brings together the community of experts, academics,
+          and activists who comprise the Creative Common affiliate network in a
+          different country every two. This year, we hop to expand our
+          invitation list including organizations and individuals
+        </p>
+
+        <p class="headline-venue abt-headline-contact-info">
+          Please contact us per Email for any further questions about CC Global
+          Summit 2015!<br>
+          <a href="#"> ccsummit2023@cckorea.org </a>
```

Comment:
[REQUIRED] Kindly make this text's color black and include an underline just like we have it in the template

Reply:
Thanks